packages/workbox-precaching/src/PrecacheController.ts

```diff
@@ -113,35 +113,37 @@ class PrecacheController {
       }
     }
 
-    const urlsToPrecache = [];
-    const urlsAlreadyPrecached = [];
+    const toBePrecached = [];
+    const alreadyPrecached = [];
```

Comment:
Looks like these two arrays take different types of objects. Maybe it'd be easier to understand what's happening if you add type information here in the declarations?

Reply:
Done.